hack/ci-e2e-kind-ha.sh

```diff
@@ -36,7 +36,7 @@ trap "
 make KUBECONFIG=$KUBECONFIG gardener-up
 
 # run test
-make KUBECONFIG=$KUBECONFIG test-e2e-local
+make KUBECONFIG=$KUBECONFIG test-e2e-local-ha
```

Comment:
```suggestion
make test-e2e-local-ha
```

not needed anymore after above suggestion